src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -150,35 +149,61 @@ export const SelectInstitutionForm = ({ onSubmit, onClose, suggestedInstitutions
               )}
             </Field>
             {values.unit?.hasPart && values.unit.hasPart.length > 0 && (
-              <Field name={SelectOrganizationFormField.Subunit}>
-                {({ field }: FieldProps<Organization>) => (
-                  <Autocomplete
-                    options={getSortedSubUnits(values.unit?.hasPart)}
-                    getOptionLabel={(option) => getLanguageString(option.labels)}
-                    renderOption={(props, option) => (
-                      <OrganizationRenderOption key={option.id} props={props} option={option} />
-                    )}
-                    onChange={(_, value) => setFieldValue(field.name, value)}
-                    filterOptions={(options, state) =>
-                      options.filter((option) =>
-                        state.getOptionLabel(option).toLocaleLowerCase().includes(state.inputValue.toLocaleLowerCase())
-                      )
-                    }
-                    renderInput={(params) => (
-                      <TextField
-                        {...params}
-                        data-testid={dataTestId.organization.subSearchField}
-                        label={t('registration.contributors.department')}
-                        variant="filled"
-                        fullWidth
+              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+                <Typography variant="h3" component="h2" sx={{ marginTop: '1rem', fontWeight: 'normal' }}>
+                  {t('common.select_unit')}
+                </Typography>
+                <Field name={SelectOrganizationFormField.Subunit}>
+                  {({ field }: FieldProps<Organization>) => (
+                    <>
+                      <Autocomplete
+                        options={getSortedSubUnits(values.unit?.hasPart)}
+                        getOptionLabel={(option) => getLanguageString(option.labels)}
+                        renderOption={(props, option) => (
+                          <OrganizationRenderOption key={option.id} props={props} option={option} />
+                        )}
+                        onChange={(_, value) => {
+                          setFieldValue(field.name, value);
+                          setSelectedSubunitId(value?.id ?? '');
```

Comment:
This is added so that when a subunit is selected from search, it automatically gets a checked bullet point (this is not the case in Advanced search)